src/_pytest/logging.py

```diff
@@ -382,20 +384,19 @@ def _finalize(self) -> None:
 
     @property
     def handler(self) -> LogCaptureHandler:
-        """Get the logging handler used by the fixture.
-
-        :rtype: LogCaptureHandler
-        """
+        """Get the logging handler used by the fixture."""
         return self._item.stash[caplog_handler_key]
 
-    def get_records(self, when: str) -> List[logging.LogRecord]:
+    def get_records(
+        self, when: "Literal['setup', 'call', 'teardown']"
```

Comment:
No need to change, but we can use `from __future__ import annotations` and get away with writing the annotations directly (instead in string form).

Reply:
Hmm it might be nice, but probably should do it for the entire code base at once?

Reply:
I think doing piecewise is fine, and of course doing all at once is fine too. But of course, just want to comment as this is fine. 👍